Solinea/goldstone-server
# logstash handling of canary events

To test, you should be able to simply:

* start  docker machine
* start RDO instance
* eval machine environment
* docker rm -f $(docker ps -qa)
* docker-compose -f docker-compose-ent-dev.yml up

Canary should generate a testflight record set every 2 minutes.  You should find them in the internal-{today} index in ES, and note that they have a type of "canary"
